foreman/requirements.txt

```diff
@@ -6,6 +6,7 @@
 #
 amqp==2.1.4               # via kombu
 billiard==3.5.0.2         # via celery
+boto==2.46.1
```

Comment:
Celery is still using boto v2.x.x for it's SQS integration. I am using boto3 for the S3 integration though!